src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/YesNo.java

```diff
@@ -36,12 +56,36 @@ public static boolean isYes(String yesNo) {
         return nonNull(yesNo) && YES.getValue().equalsIgnoreCase(yesNo);
     }
 
+    public static boolean isNo(YesNo yesNo) {
+        return NO.equals(yesNo);
+    }
+
+    public static boolean isNo(String yesNo) {
+        return NO.getValue().equalsIgnoreCase(yesNo);
```

Comment:
Feels like this should match isYes(String yesNo) above - either make this the same, or simplify isYes